build.yml

```diff
@@ -1,8 +1,8 @@
 ---
 name: "jellyfin-plugin-isomounter"
 guid: "4682DD4C-A675-4F1B-8E7C-79ADF137A8F8"
-version: "1" # Please increment with each pull request
-jellyfin_version: "10.4.0" # The earliest binary-compatible version
+version: "1"
+jellyfin_version: "10.5.0"
```

Comment:
why drop the comments? don't they work?

Reply:
[this is why](https://github.com/jellyfin/jellyfin-plugin-ldapauth/pull/30#discussion_r399961707) imo

Reply:
I disagree, at least about the comment on `jellyfin_version`. They aren't discoverable but the comments explain what those values mean.

Though it's not a blocker for sure.

Reply:
I have no opinion either way, I just remembered seeing that change done to one of my other PRs, and figured I'd do it here too, while I was in there poking.

Reply:
Actually, I had plans to fix that anyways in my plugin pull request. That field should technically be named `abi_version` or something because that's more accurate than what we have now. It could just be minimum version for now, but in the future we could potentially do things like keep compatibility tests or force the ABI versions to match exactly without having to change the manifest variable.